plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/NameEmbeddings.kt

```diff
@@ -74,6 +75,16 @@ data class ClassMemberSetter(val className: ClassName, val name: Name) : Mangled
         get() = "${className.mangled}\$setter_${name.asStringStripSpecialMarkers()}"
 }
 
+data class ExtensionPropertyGetter(override val packageName: FqName, val name: Name) : FqMangledName {
```

Comment:
Since you can have extension methods with the same name on two different types, I think you need type-based mangling here as well.  This may be easier to add once we have https://github.com/jesyspa/kotlin/pull/112

Reply:
So, should I try to integrate the modifications on overloading after merging the new PR?

Reply:
Probably easiest to do that, yes, since my changes generally make working with these things a bit simpler.  But you could implement it directly if it's little effort, too.

Reply:
Then I prefer working on the overloading having the new modifications, so I am already aligned with the latest changes